lib/tictactoe_ui.ex

```diff
@@ -1,88 +1,175 @@
 defmodule TicTacToe.UI do
 
-  def start_game() do
-    TicTacToe.make_move()
-    |> decide_turn_type()
-  end
-  
-  def get_move(game_status) do
-    print_turn(:get_player_move, game_status) 
-  end
+  @clear_screen "\e[H\e[2J"
+
+  def start_game_loop, do: start_game |> game_loop
+
+  def start_game, do: TicTacToe.first_game_status |> do_next_turn
 
-  def get_tile_selection() do
-    IO.gets("Please enter a tile selection:")
-    |> String.trim()
-    |> String.to_integer()
+  def game_loop(:game_over), do: nil
+  def game_loop(game_status) do
+    game_status  
+    |> get_move
+    |> game_loop
   end
 
-  def make_move(selected_tile, game_status) do 
-    TicTacToe.make_move(selected_tile, game_status)
+  def do_turn_type(turn_type, game_status) do
```

Comment:
Is the suffix "_type" in the function name necessary?
I feel like the first parameter 'turn_type' already conveys that information.